README.md

```diff
@@ -127,11 +139,18 @@ To run against a specific IDE, the `runIdeDirectory` property can be used as suc
 Keep in mind that the `clean` task will wipe out the content of `build/idea-sandbox`,
 so you will need to repeat some setup steps for that instance, such as configuring the JDK.
 
-Whenever you change the dependency version, the previous versions are not deleted from the sandbox, and the JVM might not load the version that you expect.
+Whenever you change the dependency version, the previous versions are not deleted from the sandbox, and the JVM might not load the version
+that you expect.
 As the `clean` task may be inconvenient, an easier workaround is to delete the jars in the sandbox, for example with:
 
     find build/idea-sandbox/ -name '*.jar' -delete
 
+Build Images
+------------
+
+The image used to build the plugin is defined in the
+`https://github.com/SonarSource/sonarlint-ci-images/blob/master/.cirrus/sonarlint_intellij_ami_tasks.star` pipeline.
```

Comment:
It's a private repository, I'm not sure it belongs in the README of a public repo